content/welcome.html

```diff
@@ -39,7 +39,7 @@ <h5 class="card-title">Press the gear icon to go to the Settings page</h5>
 			<img class="card-img-top" src="/assets/img/getting-started/step-3.png" alt="Scroll through the addons list and enable the ones that you want">
 			<div class="card-body">
 				<h5 class="card-title">Scroll through the addons list and enable the ones that you want</h5>
-				<p class="card-text">You can also configure each of the addons on the bottom section of each addons.</p>
+				<p class="card-text">You can also configure each of the addons on its bottom section.</p>
```

Comment:
"each addon" maybe?
@RedGuy12

Reply:
At the bottom of each adddon's card

Reply:
> At the bottom of each adddon's card

Can you send the full sentence? I don't understand

Reply:
`You can also configure addons at the bottom of each adddon's card`, maybe?

Reply:
> `You can also configure addons at the bottom of each adddon's card`, maybe?

The main goal is not repeat "addons", but I think this is the best ay the moment, I'll change it tomorrow.